scene/2d/ray_cast_2d.cpp

```diff
@@ -61,6 +61,21 @@ void RayCast2D::set_type_mask(uint32_t p_mask) {
 	type_mask = p_mask;
 }
 
+void RayCast2D::set_collision_mask_bit(int p_bit, bool p_value) {
+
+	uint32_t mask = get_collision_mask();
```

Comment:
I'm not sure what's the recommended style, but I think you could use the member variable directly instead of using its setter/getter, so:
```
if (p_value)
	collision_mask |= 1 << p_bit;
else
	collision_mask &= ~(1 << p_bit);
```
Same for the getter.

Reply:
But well it's implemented the same for Area2D so that's good for me :)

Reply:
I can see you merged already, do you still think it's worth to improve it?

Reply:
Oh havent seen your second comment

Reply:
I think the compiler can optimize it already anyway :P